account_financial_report/wizard/general_ledger_wizard.py

```diff
@@ -119,8 +113,8 @@ def on_change_account_range(self):
     def _init_date_from(self):
         """set start date to begin of current year if fiscal year running"""
         today = fields.Date.context_today(self)
-        last_fsc_month = self.env.user.company_id.fiscalyear_last_month
-        last_fsc_day = self.env.user.company_id.fiscalyear_last_day
+        last_fsc_month = self.env.company.fiscalyear_last_month
```

Comment:
Shouldn't this be self.company_id?

Reply:
The problem is that company_id isn't required, but maybe we could use it and fallback to self.env.company which will be more correct. I'm changing it now.